package.json

```diff
@@ -21,9 +21,11 @@
   },
   "homepage": "https://github.com/21-23/cssqd-ui#readme",
   "dependencies": {
-    "preact": "^8.2.5"
+    "preact": "^8.2.5",
+    "react-fontawesome": "^1.6.1"
```

Comment:
Not critical for now, but just curious: can we engage it with https://github.com/21-23/_qd-ui/issues/42 ?

Reply:
I'd say addressing 21-23/_qd-ui#42 will require too much effort, so let's postpone it